packages/jaeger-ui/src/components/QualityMetrics/__snapshots__/ScoreCard.test.js.snap

```diff
@@ -1,98 +1,235 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ScoreCard renders as expected when score is below max 1`] = `
-<div
-  className="ScoreCard"
->
-  <span
-    className="ScoreCard--TitleHeader"
-  >
-    Test Score
-  </span>
+<div>
   <div
-    className="ScoreCard--CircularProgressbarWrapper"
+    class="ScoreCard"
   >
-    <CircularProgressbar
-      decorationHue={120}
-      maxValue={108}
-      text="38.9%"
-      value={42}
-    />
+    <span
+      class="ScoreCard--TitleHeader"
+    >
+      Test Score
+    </span>
+    <div
+      class="ScoreCard--CircularProgressbarWrapper"
+    >
+      <svg
```

Comment:
the previous snapshot was a lot more compact, this one contains a lot of junk which is difficult to grok for a human, making it much less valuable as a tool to detect regressions. Perhaps this was the difference in using the `shallow` function - is there an equivalent in the new libs?

Reply:
Actually, the reason for that is, previously, the snapshots were containing the react-dom, but in the case of RTL, they contain HTML dom

According to official migration guide https://testing-library.com/docs/react-testing-library/migrate-from-enzyme/
The shallow function needs to be replaced with render, which seems like the only solution.

Reply:
There is a [solution on the internet](https://thesametech.com/snapshot-testing-in-rtl/#:~:text=Snapshot%20testing%20is%20often%20used,operate%20with%20the%20real%20DOM.) that suggests changing the jest snapshot serializer to output smaller snapshots.

That could break all of the tests, and thus tests would need a huge code change.